airbyte-webapp/src/pages/OnboardingPage/components/FirstSuccessfulSync/FirstSuccessfulSync.module.scss

```diff
@@ -0,0 +1,24 @@
+@use "../../../../scss/colors";
+@use "../../../../scss/variables";
+
+.container {
+  width: 100%;
+  color: colors.$white;
+  padding: 50px 78px 40px;
+  background: colors.$dark-blue-900 url("../../../../../public/stars-background.svg");
```

Comment:
:art: Given that this image is only used here, and we try to move things closer to the components, maybe you could move that `svg` into this component's folder as part of this PR and just use `url("./stars-background.svg")`. Since this is anyway handled by CRA to include this while compiling this SASS into a media folder, there is no need why this would need to be in `public` anymore.